3570/3/rating_places/db/migrate/20190729140058_add_defaults_values.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultsValues < ActiveRecord::Migration
+  def change
+    change_column :places, :rating, :float, default: 0.0
```

Comment:
If you are the only person who sees these migrations before you push it you can just change 1 of your migrations instead of producing 2